server/services/index.ts

```diff
@@ -18,6 +19,7 @@ export const services = () => {
   const userPreferenceService = new UserPreferenceService(config.apis.userPreferenceService)
   const staffLookupService = new StaffLookupService(config.apis.staffLookupService)
   const technicalUpdatesService = new TechnicalUpdatesService()
+  const cacheService = new CacheService()
```

Comment:
I don't like the name `CacheService` - it's not a cache really. Maybe `allocationStorageService`?